src/components/Home.js

```diff
@@ -1,10 +1,19 @@
 import React from 'react';
+import { Button } from 'react-bootstrap';
 
 const Home = () => {
   return (
-    <div>
-      <h1> Hello This is Home page</h1>
-    </div>
+    <main className="text-center">
+      <h1>{"CYF Quiz"}</h1>
```

Comment:
We use `{``}` for executing JavaScript inside JSX. Here, we are not running JS since we are just writing normal HTML so we should remove the curly brackets. Also, if you want to change the font size then you would do that with CSS rather than changing the heading, since using a correct heading is more important

Reply:
Oh, yes I missed that sorry.